haas/api.py

```diff
@@ -84,6 +66,7 @@ def project_delete(project):
 
     If the project does not exist, a NotFoundError will be raised.
     """
+    get_auth_backend().require_admin()
```

Comment:
I don't feel super strongly either. A possible disadvantage is that users could lock themselves out of the system, requiring an admin's assitance to get back in. And not just themselves, but other members of the project as well.

I figure just leave it as is, since neither of us seems to feel strongly.